store-api-server/src/nft/service/nft.service.ts

```diff
@@ -341,13 +341,40 @@ WHERE idx_address = $1
 `,
       [address],
     );
-    return Object.keys(PHASE2_TOKEN_ID).reduce(
-      (acc, el) => {
-        acc[el] = parseInt(keys.rows.find((r: Record<string, string>) => parseInt(r.token_id, 10) === PHASE2_TOKEN_ID[el])?.count ?? '0', 10);
-        return acc;
-      },
-      {} as Record<string, number>,
+
+    const getKeysDataRawResult = await this.conn.query(
+      `
+SELECT
+  claims.token_id AS token_id,
+  claims.token_type AS token_type,
+  mint.command->'args'->>'metadata_ipfs' AS metadata_ipfs,
+  mint.included_in AS mint_hash
+FROM claims
+JOIN peppermint.operations AS mint
+  ON  (mint.command->'args'->>'token_id')::int = claims.token_id
+  AND mint.command->>'name' = 'create_and_mint'
```

Comment:
This line should be removed - I hotfixed it out of the `release` branch of nft-claim-api a couple of days ago but hadn't put that change into `main`

Reply:
though, since we'll be running devils mints through the same peppermint instance, we need to select specifically for the token_gate contract here I think?

Reply:
Ah, that is a good point - so `AND mint.command->>'handler' = 'key'`, and we will need to put an index on that too